pkg/controller/migmigration/task.go

```diff
@@ -196,10 +200,11 @@ var FinalItinerary = Itinerary{
 		{Name: WaitForVeleroReady, Step: StepPrepare},
 		{Name: WaitForRegistriesReady, Step: StepPrepare, all: IndirectImage | EnableImage | HasISs},
 		{Name: EnsureCloudSecretPropagated, Step: StepPrepare},
-		{Name: PreBackupHooks, Step: StepBackup},
+		{Name: PreBackupHooks, Step: PreBackupHooks, all: HasPreBackupHooks},
 		{Name: CreateDirectImageMigration, Step: StepBackup, all: DirectImage | EnableImage},
 		{Name: EnsureInitialBackup, Step: StepBackup},
 		{Name: InitialBackupCreated, Step: StepBackup},
+		{Name: EnsureInitialBackupReplicated, Step: StepBackup},
```

Comment:
This should to be moved back to where it was before `EnsureFinalRestore`, because it will slow down the migration process to have the wait for replication here. The controller could be doing other work instead of blocking here. 

This change could slow down migrations by up to 1 minute (backup sync period of velero).